### PR TITLE
use seed phrase module to derive key

### PR DESCRIPTION
**Pull request by tanlethanh**

### CODE REVIEW

File: apps/web/src/state/app/authentication.ts [hunk 1]
```diff
@@ -188,16 +198,121 @@ export const storeAuthenticatedRecords = async (
 	login?: UserCredential,
 ): Promise<void> => {
 	await key.reconstructKey();
-	const privateKeys = await key.modules.privateKeyModule.getPrivateKeys();
 
 	if (login?.user) {
 		await setProfile(makeProfile(login));
 	}
 
+	// await initByPrivateKeyModule(passcode);
```
Comment (FME849) [inline apps/web/src/state/app/authentication.ts:206]: give explanation why this comment should be kept

File: apps/web/src/state/app/authentication.ts [hunk 2]
```diff
@@ -188,16 +198,121 @@ export const storeAuthenticatedRecords = async (
...
+			network: Networks.solana,
+		},
+		{
+			path: "44'/784'",
+			network: Networks.sui,
+		},
+		{
+			path: "44'/1729'",
+			network: Networks.tezos,
+		},
```
Comment (FME849) [inline apps/web/src/state/app/authentication.ts:226]: maybe a comment which explain where these `path` come from and/or how to find them in docs would be appreciated

**Review by FME849 — APPROVED** [review]

File: apps/web/src/state/app/authentication.ts (hunk 2 above)
Comment (tanlethanh) [reply apps/web/src/state/app/authentication.ts:226]: these paths are default paths for each network, we can find them in the dev docs of each network. I also noted this in [notion task](https://www.notion.so/stormgate/2a39a522cbd443a59ee2eb06ebb9dba2?v=1904b777f28648f28d21db626f9bfec4&p=83aad874791445d880937c1bee6a41ea&pm=s)

File: packages/core/utils/commonTypes.ts [hunk 3]
```diff
@@ -50,7 +50,6 @@ export enum Networks {
 	ethereum = 'ethereum',
 	solana = 'solana',
 	sui = 'sui',
-	tezos = 'tezos',
```
Comment (fluctlight-kayaba) [inline packages/core/utils/commonTypes.ts:53]: still need this I guess

Comment (tanlethanh) [reply packages/core/utils/commonTypes.ts:53]: Have been duplicated above

**Pull request merged**